apps/flows/src/app/transfer-and-pay-flow/own-account.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@angular/core';
+import { OwnAccount } from './own-account';
+import { Observable, Subject } from 'rxjs';
+import { RangeValueAccessor } from '@angular/forms';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class OwnAccountService {
+  SenderID: number;
```

Comment:
start lowercase (lower camelcase)

Reply:
https://www.techtarget.com/whatis/definition/lowerCamelCase#:~:text=What%20is%20lowerCamelCase%3F,letters%2C%20including%20the%20first%20letter.
![Screenshot 2022-11-08 at 08 23 02](https://user-images.githubusercontent.com/114976818/200500166-ca0af195-3660-401d-af63-e9bb9d65854a.png)

Reply:
So this will be `senderId`

Reply:
Fixed